### PR TITLE
Log Improvement Back Port

### LINKED ISSUE #8568: Back Port Log Improvement to 6.5

### PR DESCRIPTION
**Pull request by xzhangxian1008**
### What problem does this PR solve?

Issue Number: close #8568

Problem Summary:

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

```release-note
Change the default log level to "info" to reduce consumption of logging
```

### CODE REVIEW

File: dbms/src/Flash/Mpp/MPPTask.cpp [hunk 1]
```diff
@@ -361,14 +361,17 @@ void MPPTask::runImpl()
...
+        auto time_cost_in_preprocess_ms = stopwatch.elapsedMilliseconds();
+        LOG_DEBUG(log, "task preprocess done");
         schedule_entry.setNeededThreads(estimateCountOfNewThreads());
         LOG_DEBUG(log, "Estimate new thread count of query: {} including tunnel_threads: {}, receiver_threads: {}", schedule_entry.getNeededThreads(), dag_context->tunnel_set->getExternalThreadCnt(), new_thread_count_of_mpp_receiver);
 
         scheduleOrWait();
 
-        LOG_INFO(log, "task starts running");
+        auto time_cost_in_schedule_ms = stopwatch.elapsedMilliseconds() - time_cost_in_preprocess_ms;
+        LOG_INFO(log, "task starts running, time cost in schedule: {} ms, time cost in preprocess", time_cost_in_schedule_ms, time_cost_in_preprocess_ms);
```
Comment (SeaRise) [inline dbms/src/Flash/Mpp/MPPTask.cpp:374]:
```suggestion
        LOG_INFO(log, "task starts running, time cost in schedule: {} ms, time cost in preprocess {} ms", time_cost_in_schedule_ms, time_cost_in_preprocess_ms);
```

File: dbms/src/Flash/Mpp/MinTSOScheduler.cpp [hunk 2]
```diff
@@ -235,7 +235,14 @@ bool MinTSOScheduler::scheduleImp(const UInt64 tso, const MPPQueryTaskSetPtr & q
             GET_METRIC(tiflash_task_scheduler, type_waiting_queries_count).Set(waiting_set.size());
             GET_METRIC(tiflash_task_scheduler, type_waiting_tasks_count).Increment();
         }
-        LOG_INFO(log, "threads are unavailable for the query {} or active set is full (size =  {}), need {}, but used {} of the thread soft limit {},{} waiting set size = {}", tso, active_set.size(), needed_threads, estimated_thread_usage, thread_soft_limit, isWaiting ? "" : " put into", waiting_set.size());
+        LOG_INFO(log, "Resource temporary not available for start_ts {}(is first schedule: {}), available threads count are {}, available active set size = {}, "
```
[the remark this answers is not in the document]
Comment (xzhangxian1008) [reply dbms/src/Flash/Mpp/MinTSOScheduler.cpp]: > query with

done

File: dbms/src/Flash/Mpp/MPPTask.cpp (hunk 1 above)
Comment (xzhangxian1008) [reply dbms/src/Flash/Mpp/MPPTask.cpp]: > 

done

**Review by SeaRise — APPROVED** [review]
lgtm

**Review by windtalker — APPROVED** [review]
LGTM

**Pull request merged**